docs/Fields.md

```diff
@@ -477,43 +466,22 @@ export default TextField;
 
 ## Adding A Label To Custom Field Components
 
-When you use one of the react-admin `Field` components in an `Edit`, `Create` or `Show` view, react-admin includes a label on top of the field value, as in the following example:
-
-![field labels](./img/field-addlabel.png)
-
-For your custom fields, however, the label doesn't appear by default. You need to opt-in to this feature by setting the `addLabel` prop to `true` in the `defaultProps`.
+If you want to have a label for your custom fields in an `Edit`, `Create` or `Show` view, you need to wrap your component inside a `<Labeled>` component:
```

Comment:
No in a Show view. Besides, I think this tip is misleading. It's not the responsibility of the field anymore to add a label.